ScatterPeas/scatterpeas/__init__.py

```diff
@@ -0,0 +1,47 @@
+from pyramid.config import Configurator
+from sqlalchemy import engine_from_config
+from pyramid.authentication import AuthTktAuthenticationPolicy
+from pyramid.authorization import ACLAuthorizationPolicy
+import os
+from models import ReminderFactory, groupfinder, RootFactory, UserFactory, AliasFactory
```

Comment:
These imports could be part of the list of things imported from `.models` below.